src/EnergyPlus/UnitarySystem.cc

```diff
@@ -1691,10 +1687,13 @@ namespace UnitarySystems {
             SizingMethod = DataHVACGlobals::HeatingAirflowSizing;
             // SizingString = UnitarySystemNumericFields(UnitarySysNum).FieldNames(FieldNum) + " [m3/s]";
             TempSize = this->m_MaxHeatAirVolFlow;
-            SaveCurDuctType = DataSizing::CurDuctType;
-            DataSizing::CurDuctType = DataHVACGlobals::Heating;
             if ((HeatingSAFlowMethod == SupplyAirFlowRate) || (HeatingSAFlowMethod == None)) {
                 ReportSizingManager::RequestSizing(CompType, CompName, SizingMethod, SizingString, TempSize, PrintFlag, RoutineName);
+                if (DataSizing::CurSysNum > 0 && DataSizing::CurOASysNum == 0) {
+                    if (this->m_HeatingCoilType_Num == DataHVACGlobals::Coil_HeatingWater &&
+                        DataSizing::FinalSysSizing(DataSizing::CurSysNum).SysAirMinFlowRat > 0.0)
+                        TempSize *= DataSizing::FinalSysSizing(DataSizing::CurSysNum).SysAirMinFlowRat;
+                }
```

Comment:
For water heating coil sizing the heating flow rate is needed as main duct flow * SysAirMinFlowRat